### ISSUE TITLE
Always Show the Next and Prev Controls

### ISSUE THREAD

**Issue by mkhatib** [revolunet/angular-carousel#298]
Is there a way to configure the carousel to always keep the perv arrow even on the first slide and clicking it will cycle back to the last slide?

**Comment by dweave** [comment]
Hey not sure it's 'configurable' but certainly doable.  just create your own next/prev buttons and have them increment/decrement the index.  When new index becomes greater than slides.length index should become zero.  When index is 0 and prev is pressed, new index becomes slides.length - 1.  

If this solution is obvious and your looking for a truly configurable solution, I'm sorry, but thought i'd offer my 2 cents.

**Comment by mkhatib** [comment]
Thanks @dweave that was easier than I thought it would be. I didn't actually had to increase or decrease the length myself. All I needed to do is to manually create the elements without the ng-if attributes to always show them. angular-carousel took care of figuring out this is the last or first slide and did the right thing.

All I had to do is insert this after the last slide and remove the rn-carousel-control attribute on the main carousel container:

      <div class="rn-carousel-control">
        <span class="rn-carousel-control rn-carousel-control-prev" ng-click="prevSlide()"></span>
        <span class="rn-carousel-control rn-carousel-control-next" ng-click="nextSlide()"></span>
      </div>
